src/middleware/log-request-errors.ts

```diff
@@ -1,4 +1,9 @@
-module.exports = (err, req, res, next) => {
-  req.log.error(err)
+import {NextFunction} from 'express'
+import {Request,Response} from './logging'
```

Comment:
It'd be great if we could figure out a better way for `logging` to extend the built-in express types so we don't have to do this.

Reply:
@bkeepers you mean the imports from two different places?

Reply:
Yeah, I just mean `logging` having to export `{Request, Response}`, which are really from express with some special sauce.

Reply:
I updated this PR to do this.